test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -13,7 +13,7 @@ public class LocalizationTest
     {
         private static readonly string _applicationPath =
 #if DNX451
-            Path.GetFullPath(Path.Combine("..", "..", "..", "LocalizationWebsite"));
+            Path.GetFullPath(Path.Combine("..", "..", "..", "..", "test", "LocalizationWebsite"));
```

Comment:
Does https://github.com/aspnet/Localization/blob/dev/test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs#L16 need to be updated?